src/adapters/ReactThirteenAdapter.js

```diff
@@ -0,0 +1,215 @@
+  createMountRenderer(options) {
+    const domNode = options.attachTo || global.document.createElement('div');
+    let instance = null;
+    return {
+      render(el/* , context */) {
+        const wrappedEl = React.createElement(SimpleWrapper, {
+          node: el,
+        });
+        instance = React.render(wrappedEl, domNode);
+      },
+      unmount() {
+        React.unmountComponentAtNode(domNode);
+      },
+      getNode() {
+        return instanceToTree(instance._reactInternalInstance._renderedComponent);
+      },
+      simulateEvent(node, event, mock) {
+        const mappedEvent = mapNativeEventNames(event);
+        const eventFn = TestUtils.Simulate[mappedEvent];
+        if (!eventFn) {
+          throw new TypeError(`ReactWrapper::simulate() event '${event}' does not exist`);
+        }
+        // eslint-disable-next-line react/no-find-dom-node
+        eventFn(React.findDOMNode(node.instance), mock);
+      },
+      batchedUpdates(fn) {
+        return batchedUpdates(fn);
+      },
+    };
+  }
+
+  createShallowRenderer(/* options */) {
+    const renderer = createShallowRenderer();
+    let isDOM = false;
+    let cachedNode = null;
+    return {
+      render(el, context) {
+        cachedNode = el;
+        /* eslint consistent-return: 0 */
+        if (typeof el.type === 'string') {
+          isDOM = true;
+        } else {
+          isDOM = false;
+          return renderer.render(el, context); // TODO: context
+        }
+      },
+      unmount() {
+        renderer.unmount();
+      },
+      getNode() {
+        if (isDOM) {
+          return elementToTree(cachedNode);
+        }
+        const output = renderer.getRenderOutput();
+        return {
+          nodeType: 'class',
+          type: cachedNode.type,
+          props: cachedNode.props,
+          instance: renderer._instance._instance,
+          rendered: elementToTree(output),
+        };
+      },
+      simulateEvent(node, event, ...args) {
+        const handler = node.props[propFromEvent(event)];
+        if (handler) {
+          withSetStateAllowed(() => {
+            // TODO(lmr): create/use synthetic events
+            // TODO(lmr): emulate React's event propagation
+            batchedUpdates(() => {
+              handler(...args);
+            });
+          });
+        }
+      },
+      batchedUpdates(fn) {
+        return withSetStateAllowed(() => batchedUpdates(fn));
+      },
+    };
+  }
+
+  createStringRenderer(/* options */) {
+    return {
+      render(el /* , context */) {
+        return React.renderToStaticMarkup(el);
+      },
+    };
+  }
+
+  // Provided a bag of options, return an `EnzymeRenderer`. Some options can be implementation
+  // specific, like `attach` etc. for React, but not part of this interface explicitly.
+  // eslint-disable-next-line class-methods-use-this, no-unused-vars
+  createRenderer(options) {
+    switch (options.mode) {
+      case 'mount': return this.createMountRenderer(options);
```

Comment:
It'd create a single source of truth for these strings; whereas otherwise, the string "mount" might need to appear more than once in a codebase. I'm a strong proponent of a string never appearing in a codebase more than once, if possible.